protocol/testing/containertest/preupgrade_genesis.json

```diff
@@ -1685,6 +1685,16 @@
         "min_num_votes_per_sample": 15
       }
     },
+    "marketmap": {
```

Comment:
Adding this because it seems newly added modules are having a genesis added (like revshare)

Reply:
This is the pre-upgrade genesis, e.g. the genesis state for the chain before the upgrade, so it should not have a genesis added here.

Reply:
Is the revshare module also being added?  I think it has an entry here which should be removed